.env.example

```diff
@@ -45,8 +46,8 @@ NODE_ENABLE_NODE_API=true
 NODE_API_PORT=9091
 
 # goerli smart contracts
-NODE_EIGENDA_SERVICE_MANAGER=0x9FcE30E01a740660189bD8CbEaA48Abd36040010
-NODE_BLS_OPERATOR_STATE_RETRIVER=0x737Dd62816a9392e84Fa21C531aF77C00816A3a3
+NODE_EIGENDA_SERVICE_MANAGER=0xa3b1689Ab85409B15e07d2ED50A6EA9905074Ee5
```

Comment:
@ian-shim need a second set of eyes to confirm these new ones are correct.

Reply:
Yes, that looks correct